### ISSUE TITLE
Improve article 2012.05155 - hovering/clicking on citations doesn't seem to work

### ISSUE THREAD

**Issue by pelegm** [dginev/ar5iv#34]
**Exact location of issue**
For example, in the first section of https://ar5iv.org/html/2012.05155.

**Problem details**
I expected to see something when hovering (or clicking) the reference `[EFLS95]` (for example), but nothing happens.

**Desktop**
 - OS: Ubuntu 20.04
 - Browser: Firefox 96.0

**Comment by dginev** [comment]
Right, if you go to the bottom you can confirm the entire References section is empty.

**Comment by pelegm** [comment]
Is it a problem with translating biblatex perhaps?

**Comment by dginev** [comment]
If it's using biblatex, then certainly yes. We have a tracking issue [373](https://github.com/brucemiller/LaTeXML/issues/373) in latexml for it, and it needs a dedicated binding. 

A matter of it swimming up to the top of the "new binding queue" - which will be soon, as it is a widely used missing package in arXiv.

**Comment by pelegm** [comment]
I am. Shall I close this issue?

**Comment by dginev** [comment]
Why would you? My expectation for the issues in this repository is that they stay open until the article in question is properly upgraded and corrected on the live ar5iv site. 

They are not duplicates to the more technical latexml-specific issues in that repository. Instead, the issues here provide examples and tests over which latexml should improve.

**Comment by pelegm** [comment]
Update and correction:
1. References stil don't show (in most of my papers...)
2. I'm using `bibtex` (and not `biblatex`)

**Comment by pelegm** [comment]
Judging by a comment in #485, I believe the problem might be related to `amsrefs`.
